chinglee-iot/FreeRTOS
# cover all branches within prvAddTaskToReadyList

cover all branches within prvAddTaskToReadyList as called by several APIs

Description
-----------
Create additional tests to cover the case where prvAddTaskToReadyList is called when the
task being added to the ready list has a priority less than, or greater than the current action priority.

Test Steps
-----------
cd FreeRTOS/Test/CMock
make coverage

Related Issue
-----------
TS-24208
TS-24219
TS-24213
TS-24214




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
